noir/noir-repo/tooling/acvm_cli/src/cli/mod.rs

```diff
@@ -0,0 +1,48 @@
+use clap::{Parser, Subcommand};
+use color_eyre::eyre;
+use const_format::formatcp;
+
+mod execute_cmd;
+mod fs;
+
+const GIT_HASH: &str = env!("GIT_COMMIT");
+const IS_DIRTY: &str = env!("GIT_DIRTY");
+const NARGO_VERSION: &str = env!("CARGO_PKG_VERSION");
+
+static VERSION_STRING: &str = formatcp!(
+    "version = {}\n(git version hash: {}, is dirty: {})",
+    NARGO_VERSION,
```

Comment:
```suggestion
    ACVM_VERSION,
```

Got to change above as well